src/components/KTreeList/KTreeDraggable.vue

```diff
@@ -184,6 +200,16 @@ const handleSelectionEvent = (item: TreeListItem): void => {
   emit('selected', item)
 }
 
+const childrenVisibilityMap = ref(new Map<string, boolean>())
+
+const handleExpandedEvent = (value: boolean, id: string): void => {
+  childrenVisibilityMap.value.set(id, value)
```

Comment:
nit: can you rename the first param something other than "value" since that's already used everywhere in Vue?